src/luascript.h

```diff
@@ -377,6 +378,12 @@ class LuaScriptInterface
 		lua_setfield(L, -2, index);
 	}
 
+	static void setField(lua_State* L, const std::string index, std::string value)
+	{
+		pushString(L, value);
+		lua_setfield(L, -2, index.c_str());
+	}
+
```

Comment:
If you make `index` and `value` a `std::string_view` you can merge this declaration with the previous one `void setField(lua_State* L, const char* index, const std::string& value)`